pyproject.toml

```diff
@@ -66,6 +66,10 @@ python-multipart = ">=0.0.6"
 better-devtools = "^0.13.3"
 pytest-sugar = "^1.0.0"
 
+[tool.poetry.group.tutorial.dependencies]
+fastapi-cli = ">=0.0.5"
+uvicorn = ">=0.31.0"
```

Comment:
Poetry groups are strictly for internal contributors -- they are not actually getting published to PyPi so our users will not be able to install them.

Instead, such stuff is done using [extras](https://python-poetry.org/docs/pyproject#extras). We can even migrate to [uv](https://docs.astral.sh/uv/concepts/dependencies/#optional-dependencies) as poetry's way of handling these dependencies is rather lacking (though we could implement the whole thing with poetry). 

In addition, I propose to use the naming from fastapi: `standard`.

Reply:
Thank you for the feedback, I will correct that. Installing fastapi[standard] instead of just fastapi-cli and uvicorn separately would be a more comprehensive approach if we use standard. What do you think about that ?

I've been enjoying uv recently and I will try to look forward to migrating to it.

Reply:
> Installing fastapi[standard] instead of just fastapi-cli and uvicorn separately would be a more comprehensive approach if we use standard. What do you think about that ?

I think, the solutions are nearly equivalent here so I'd be happy with either 👍